probe-rs/src/bin/probe-rs/cmd/dap_server/server/debugger.rs

```diff
@@ -728,15 +728,15 @@ impl Debugger {
         });
 
         download_options.progress = flash_progress;
-
+        let target = session_data.session.target().clone();
         let flash_result = download_file_with_options(
             &mut session_data.session,
             path_to_elf,
             self.config
                 .flashing_config
                 .format_options
                 .clone()
-                .into_format()?,
+                .into_format(&target)?,
```

Comment:
Can we construct this before the function call and avoid the clone? :)

Reply:
Good point! Done :)